input.lisp

```diff
@@ -288,14 +302,18 @@ match with an element of the completions."
                                                :background (xlib:gcontext-foreground gcontext))
                    (draw-image-glyphs win gcontext (screen-font screen)
                                       x
-                                      (font-ascent (screen-font screen))
+                                      (* (+ *message-window-padding*
+                                            (font-ascent (screen-font screen)))
+                                         prompt-lines-length)
                                       (string char)
                                       :translate #'translate-id
                                       :size 16))
             else
               do (draw-image-glyphs win gcontext (screen-font screen)
                                     x
-                                    (font-ascent (screen-font screen))
+                                    (* (+ *message-window-padding*
+                                          (font-ascent (screen-font screen)))
+                                       prompt-lines-length)
```

Comment:
That is far enough repetition to warrant a separate function `text-height`.

Reply:
abstract this into a function and update the calls to it and I'll merge this PR.

Reply:
I guess I need to rebase on top of #404 too, which should solve the repetition issue.